pkgs/by-name/fi/finamp/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  flutter,
+  mpv,
+  patchelf,
+  fetchFromGitHub,
+  makeDesktopItem,
+}:
+let
+  version = "0.9.7-beta";
+in
+flutter.buildFlutterApplication {
+  inherit version;
+  pname = "finamp";
+  src = fetchFromGitHub {
+    owner = "jmshrv";
+    repo = "finamp";
+    rev = version;
+    hash = "sha256-+72zjIfLxhFQ2JmNGsQthVFpFRlYXs97/p31hr9h/eU=";
+  };
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  nativeBuildInputs = [ patchelf ];
+  buildInputs = [ mpv ];
+
+  gitHashes = {
+    media_kit_libs_windows_audio = "sha256-p3hRq79whLFJLNUgL9atXyTGvOIqCbTRKVk1ie0Euqs=";
+    balanced_text = "sha256-lSDR5dDjZ4garRbBPI+wSxC5iScg8wVSD5kymmLbYbk=";
+    isar_generator = "sha256-lWnHmZmYx7qDG6mzyDqYt+Xude2xVOH1VW+BoDCas60=";
+    palette_generator = "sha256-mnRJf3asu1mm9HYU8U0di+qRk3SpNFwN3S5QxChpIA0=";
+    split_view = "sha256-unTJQDXUUPVDudlk0ReOPNYrsyEpbd/UMg1tHZsmg+k=";
+  };
+
+  postFixup = ''
+    patchelf $out/app/finamp --add-needed libisar.so --add-needed libmpv.so --add-rpath ${lib.makeLibraryPath [ mpv ]}
+  '';
+
+  postInstall = ''
+    d=$out/share/icons/hicolor/scalable/apps
+    mkdir -p $d
+    cp $src/assets/icon/icon_foreground.svg $d/finamp.svg
+
+    mkdir -p $out/share/applications
+    cp $desktopItem/share/applications/*.desktop $out/share/applications
+    substituteInPlace $out/share/applications/*.desktop --subst-var out
+  '';
+
+  desktopItem = makeDesktopItem {
+    name = "Finamp";
+    exec = "@out@/bin/finamp";
```

```suggestion
    exec = "finamp";
```

I think you can just use `"finamp"` here, since upon installation it will be in the PATH.

This would also remove line 45 (`substituteInPlace`).

---

I changed it and removed the call to `substituteInPlace`